### ISSUE TITLE
Hyper starts bu does not show command prompt

### ISSUE THREAD

**Issue by karsibali** [zeit/hyper#4465]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Ubuntu 18.04 LTS
- **Hyper.app version**: hyper-3.0.2-x86_64.AppImage
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/karsibali/624b5901f79b11f94fb4a8c1a401573f
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: Is Vanilla
## Issue

I am using the hyper AppImage on a Ubuntu 18.04 LTS system.
When I start the hyper terminal I see the hyper window but the command line is not displayed as seen on the screenshot given below:

![Screenshot from 2020-04-10 01-49-58](https://user-images.githubusercontent.com/6684437/78947283-9aaaf180-7acd-11ea-90ac-5bc29bd3aaa4.png)

If I press Ctrl-C then the command line shows up like this:
![Screenshot from 2020-04-10 01-53-05](https://user-images.githubusercontent.com/6684437/78947464-14db7600-7ace-11ea-9ed8-4bb0e544b300.png)

I listed the running process tree before hitting Ctrl-C. Here is the screenshot of htop output:

![Screenshot from 2020-04-10 01-52-44](https://user-images.githubusercontent.com/6684437/78947543-4bb18c00-7ace-11ea-9bf4-127e1dcc0e34.png)

As you can see there are three sub-processes spawn by hyper. These three processes are blocking hyper to start completely. After I hit Ctrl-C these three processes get killed and hyper then displays the command line. The three processes that use high CPU and block hyper are:
```
cat /dev/urandom
tr -dc a-zA-Z0-9
fold -w 24
```

**Comment by LabhanshAgrawal** [comment]
Hyper seems to have started the `/bin/bash --login` process and then bash spawns those blocking processes, you should check your `.bash_profile` and `.bashrc` to see if these are started from there.

**Comment by karsibali** [comment]
It turned out to be McFly reverse history script. When I comment out the mcfly script from my .bashrc file Hyper works OK.
What I don't understand here is, normally this is not a blocking process the line in the mcfly.bash script, which spawns these processes is as follows:
```
export MCFLY_SESSION_ID=$(cat /dev/urandom | env LC_ALL=C tr -dc 'a-zA-Z0-9' | fold -w 24 | head -n 1)
```
This should normally end without any problem. But for whatever reason the last part of the command (head -n 1) does not work. So it becomes a blocking process.
Do you have any idea?

**Comment by LabhanshAgrawal** [comment]
Looks like a known issue in McFly, it's closed so maybe you need to update it?
https://github.com/cantino/mcfly/issues/32
https://www.unix.com/unix-for-advanced-and-expert-users/271649-cmd-cat-dev-urandom-not-closing-cleanly.html

**Comment by LabhanshAgrawal** [comment]
as the problems seems to be unrelated to hyper, you can close this issue

**Comment by karsibali** [comment]
Thanks a lot...

**Comment by LabhanshAgrawal** [comment]
@karsibali 
What ended up being the solution for your McFly issue?

**Comment by karsibali** [comment]
I left a comment on McFly github page. There is no comments on that yet.
Currently I simply I disabled McFly from my .bashrc file.

**Comment by karsibali** [comment]
I upgraded McFly to the latest version. It seems to be fixed with this version (v0.3.5).
